pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "wtprompt"
-version = '0.1.1'
+version = '0.1.11'
```

Comment:
Why the gap? If you want to signal that this is a bigger change than bugfixes just move to `0.2.0` I'd say.